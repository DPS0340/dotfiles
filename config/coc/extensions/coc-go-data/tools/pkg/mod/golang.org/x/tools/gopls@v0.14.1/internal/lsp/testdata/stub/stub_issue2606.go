package stub

type I interface{ error }

type C int

var _ I = C(0) //@suggestedfix("C", "quickfix", "")
