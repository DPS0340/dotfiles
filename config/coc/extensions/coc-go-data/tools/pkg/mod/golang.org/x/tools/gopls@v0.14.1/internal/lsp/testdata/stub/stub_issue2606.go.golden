-- suggestedfix_stub_issue2606_7_11 --
package stub

type I interface{ error }

type C int

// Error implements I.
func (C) Error() string {
	panic("unimplemented")
}

var _ I = C(0) //@suggestedfix("C", "quickfix", "")

